trl/trainer/grpo_trainer.py

```diff
@@ -213,6 +260,12 @@ def reward_func(completions, **kwargs):
                 - A custom reward function: The function is provided with the prompts and the generated completions,
                   plus any additional columns in the dataset. It should return a list of rewards. For more details, see
                   [Using a custom reward function](#using-a-custom-reward-function).
+                  
+                  Custom reward functions can also return None for the entire function call or for individual samples
```

Comment:
```suggestion
                  Custom reward functions can also return None
```

I'd recommend not supporting returning None for the entire batch, I complicates things too much imo